### PR TITLE
Add topics cleaning and tagging code

### PR DESCRIPTION
**Pull request by sara-02**

### CODE REVIEW

**Comment by sara-02** [comment]
@miteshvp PTAL

File: topic_cleaning/aliases.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+	"ai": "artificial-intelligence",
+	"analytics": "data-analysis",
+	"ananlyis": "data-analysis",
```
Comment (jpopelka) [inline topic_cleaning/aliases.json]: typo in "ananlyis"

File: topic_cleaning/aliases.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  	"webserver": "web-server",
+  	"amazon": "aws",
+  	"json-serializer": "json-serialization",
+  	"hashmap": "hash-map",
+  	"crypto": "cryptography",
+  	"operations": "operation",
+  	"datetime": "date-time",
+  	"ip-address": "ip",
+  	"visualize": "visualization",
+  	"visualise": "visualization",
```
Comment (jpopelka) [inline topic_cleaning/aliases.json:40]: duplicate

File: topic_cleaning/aliases.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  	"worker": "workers",
+  	"protocols": "protocol",
+  	"scikit-learn-api": "scikit-learn",
+  	"scikitlearn": "scikit-learn",
+  	"plotting": "plots",
+  	"plot": "plots",
+  	"data-base": "database",
+  	"timezone":"timezones",
+  	"deserialization": "deserialise", 
+  	 "deserialisation": "deserialise"
```
Comment (jpopelka) [inline topic_cleaning/aliases.json]: duplicate

File: topic_cleaning/topic_cleaner.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        self.stop_words = Cleaner.load_stop_words() #set of stop_words
+        self.aliases = Cleaner.load_aliases() # dict of aliases    
+
+    @staticmethod
+    def load_stop_words():
+        stop_words = []
+        with open('stop_words.txt', 'r') as f:
+            for line in f.readlines():
+                stop_words.append(line[:-1]) # to remove \n at the end of words
+        return set(stop_words)
```
Comment (jpopelka) [inline topic_cleaning/topic_cleaner.py]: You can directly add to set:

```python
stop_words = set()
stop_words.add(line[:-1])
```

File: topic_cleaning/aliases.json (hunk 2 above)
Comment (sara-02) [reply topic_cleaning/aliases.json:40]: @jpopelka that is saved on purpose. I don't know how to let the the system understand that with different spelling, they still mean the same. So for some time, these duplicates will be present.

Comment (jpopelka) [reply topic_cleaning/aliases.json:40]: oh, sorry, I didn't notice it's actually different

File: topic_cleaning/topic_cleaner.py [hunk 5]
```diff
@@ -15,11 +15,11 @@ def __init__(self):
 
     @staticmethod
     def load_stop_words():
-        stop_words = []
+        stop_words = set
```
Comment (jpopelka) [inline topic_cleaning/topic_cleaner.py]: `set()`

**Comment by sara-02** [comment]
@jpopelka Fixed the set(), had to add another s3 datastore in addition to graph.

**Comment by sara-02** [comment]
Not needed, the work flow has been replicated in the server and worker repo.

**Pull request closed without merging**